Mimir.Worker/Queries.graphql

```diff
@@ -26,10 +26,11 @@ query GetPatchTableTransactions($blockIndex: Long!) {
     }
 }
 
-query GetTransactionSigners($blockIndex: Long!) {
+query GetTransactionSigners($blockIndex: Long!, $limit: Long!) {
     transaction {
-        ncTransactions(startingBlockIndex: $blockIndex, limit: 1, actionType: "^[a-zA-Z0-9]*$") {
+        ncTransactions(startingBlockIndex: $blockIndex, limit: $limit, actionType: "^.*$") {
             signer
+            serializedPayload
```

Comment:
These changes are to specify changed states' addresses' type. And I think we can know them from action data. Actually, in the GetDerivedAddresses code, it's like this. Then how about getting actions directly instead to get entire transaction payload? 🤔

Reply:
umm, how to get the actions data?

Reply:
You may be using `query.transaction.ncTransactions`. Then you can use `actions.raw` field like the below:

```
query {
  transaction {
    ncTransactions {
      actions {
        raw
      }
    }
  }
}
```

Reply:
Thanks. Going forward, it looks like we just need the action values to track down the addresses